### PR TITLE
feat: notify with build url

### LINKED ISSUE #105: Add a direct link to the build in the GitHub notification
ETQU, Instead of being redirecting to the homepage of argos, I'm redirected to the specific build.

### Motivation

It saves time for users.

### Detail

We gonna need a path builder, we could use rails as inspiration.

### PR DESCRIPTION
**Pull request by neoziro**
Closes #105

Also dramatically speed up test by using delete instead of truncate (x4 speed up).

### CODE REVIEW

File: config/test.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "server": {
     "logFormat": "",
+    "url": "http://www.argos-ci.test"
```
Comment (oliviertassinari) [inline config/test.json]: why not using `url`?

File: src/modules/urls/buildUrl.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import config from 'config'
+
+export async function formatUrlFromBuild(build, { absolute = false } = {}) {
```
Comment (oliviertassinari) [inline src/modules/urls/buildUrl.js]: Shouldn't we make that module cross environment (browser/node)? I see some places where I could use it on the front.

Comment (neoziro) [inline src/modules/urls/buildUrl.js]: A little complicated, but why not :)

File: config/test.json (hunk 1 above)
Comment (neoziro) [inline config/test.json]: It's cleaner I think, the dev url is not very nice and it's not relevant.

Comment (oliviertassinari) [inline config/test.json]: I mean, the use case it the same. Aren't we introducing a duplicate?

**Pull request merged**